### ISSUE TITLE
Scrollspy doesn't work with CJK and Greek languages anchor links.

### ISSUE THREAD

**Issue by razonyang** [razonyang/hugo-theme-bootstrap#651]
**Describe the bug**

```
Uncaught DOMException: Failed to execute 'querySelector' on 'Element': 
'#%E8%AA%BF%E6%95%B4%E5%9C%96%E7%89%87%E5%A4%A7%E5%B0%8F' is not a valid selector.
```

twbs/bootstrap#36939

**Comment by iamgiannisx** [comment]
The same bug exists with Greek language.

**Comment by razonyang** [comment]
This won't be fixed, see twbs/bootstrap#37545.
